run-test

```diff
@@ -1,13 +1,35 @@
 #!/bin/bash -e
+# For coverage tests, run as
+#     coverage erase
+#     PATH=$PWD:$PATH RUNNER="coverage run --include=$PWD/**/*" ./run-test
+#     coverage html
+# See ./run-coverage
+RUNNER="${RUNNER:-python2}"
 
-rm -rf temp 
-ocropus-nlbin tests/testpage.png -o temp
-ocropus-gpageseg 'temp/????.bin.png'
-ocropus-rpred -n 'temp/????/??????.bin.png'
-ocropus-hocr 'temp/????.bin.png' -o temp.html
-ocropus-visualize-results temp
-ocropus-gtedit html temp/????/??????.bin.png -o temp-correction.html
+test_page() {
+    set -x
+    $RUNNER ocropus-nlbin tests/testpage.png -o temp
+    $RUNNER ocropus-gpageseg 'temp/????.bin.png'
+    $RUNNER ocropus-rpred -n 'temp/????/??????.bin.png'
+    $RUNNER ocropus-hocr 'temp/????.bin.png' -o temp.html
+    $RUNNER ocropus-visualize-results temp
```

Comment:
That warning is unrelated but something to investigate. It's another rounding issue like #123